packages/localizations/src/en-US.ts

```diff
@@ -281,6 +305,7 @@ export const enUS: LocalizationResource = {
       },
     },
     navbar: {
+      apiKeys: 'API keys',
```

Comment:
Sometimes "keys" is capitalized, sometimes not. Is this on purpose?

Reply:
I think for this it should be capital since the pages use capitalized "keys". Updated!